docs/narr/cookiecutters.rst

```diff
@@ -3,7 +3,16 @@
 Pyramid cookiecutters
 =====================
 
+.. versionadded:: 1.8
+    Added cookiecutter support.
+
 .. versionadded:: 1.10
```

Comment:
s/b `versionchanged`

Reply:
Wrong one. We added support in 1.8, changed it in 1.10.

Reply:
Ah, indeed. Sorry about that, I guess find and replace only really works when you read it first.

Reply:
Meh, who has time to read? 😝 

I'll start a full final review tonight.